app/src/main/java/com/example/criminalintent/presentation/auth/signup/SignUpFragment.kt

```diff
@@ -86,8 +78,7 @@ class SignUpFragment : Fragment() {
             onSignUpButtonPressed()
         }
         loginTextView.setOnClickListener {
-            callbacks?.alreadyHaveAnAccount()
-            activity?.supportFragmentManager?.popBackStack()
+            navController.popBackStack()
```

Comment:
I don't think that relying on the fact that currently the only parent of this is login fragment is a good idea. In the future we may have more entry points to this.
I think it's best to specify the login fragment exactly and use `popUpTo`: https://developer.android.com/guide/navigation/navigation-navigate#pop

This way, the navigation is shown in the nav graph as well